samples/complex_app/app.py

```diff
@@ -39,7 +39,7 @@ async def get_webhook_details(domain_name: DomainName) -> WebhookData:
 app = SaleorApp(
     validate_domain=validate_domain,
     save_app_data=store_app_data,
-    webhook_handlers=webhook_handlers,
+    http_webhook_handlers=webhook_handlers,
```

Comment:
@pkucmus As I wrote you on PW. I would consider if this is the best way for handling this. IMO we could still have `webhook_handler` and add settings with choices `http` and `sqs`. We don't support subscription for more than one webhook, so I would keep it simple and use one argument for webhook handlers for both interfaces. Does it make sense?